libpod/top.go

```diff
@@ -0,0 +1,37 @@
+package libpod
+
+import (
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/projectatomic/libpod/utils"
+	"github.com/sirupsen/logrus"
+)
+
+// This can be removed is stats is merged
+var CGroupParent = "/libpod_parent"
+
+// GetContainerPids reads sysfs to obtain the pids associated with the container's cgroup
+func (c *Container) GetContainerPids() ([]string, error) {
+	taskFile := filepath.Join("/sys/fs/cgroup/pids", CGroupParent, fmt.Sprintf("libpod-conmon-%s", c.ID()), c.ID(), "tasks")
+	logrus.Debug("reading pids from ", taskFile)
+	content, err := ioutil.ReadFile(taskFile)
+	if err != nil {
+		return []string{}, errors.Wrapf(err, "unable to read pids from %s", taskFile)
+	}
+	return strings.Split(string(content), "\n"), nil
+}
+
+// GetContainerPidInformation calls ps with the appropriate options and returns
+// the results as a string
+func (c *Container) GetContainerPidInformation(args []string) (string, error) {
+	logrus.Debug("Executing: ", strings.Join(args, " "))
+	results, err := utils.ExecCmd(args[0], args[1:]...)
```

Comment:
This says it calls ps, but it calls args[0] - can we hardcode ps here?

Reply:
yup